zone/groups.cpp

```diff
@@ -1141,7 +1146,7 @@ bool Group::LearnMembers() {
 		return false;
 	}
 
-	int memberIndex = 0;
+	int memberIndex = 1;	//starts at 1 becasuse leader [0] is done specifically
```

Comment:
Because

Reply:
Update the comment to reflect the split of group table structure.  Position 0 is in group_leaders, while 1-5 is in group_id.  I am not sure why it is this way though.  :-)

Reply:
no longer needed, so cleaned up